android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/AuthParamsWrapper.kt

```diff
@@ -7,6 +7,7 @@ class AuthParamsWrapper(
     val environment: String,
     val dbDirectory: String?,
     val historySyncUrl: String?,
+    val customLocalUrl: String?,
```

Comment:
FWIW, we use `apiUrl` in the browser and node SDKs. it overrides the `env` options.

https://github.com/xmtp/xmtp-js/blob/main/sdks/node-sdk/src/Client.ts#L56-L60

Reply:
This is in the bridge so doesn't actually get exposed or called anywhere. The way we have this setup in Android and iOS we append the http and the port. So it's not a url but the host that we allow being set to custom. Maybe in the future we can modify that but for now leaving as is.